src/components/teampage/team-members/teams-styles.js

```diff
@@ -0,0 +1,72 @@
+import styled from "styled-components"
+import hex from "../../../images/blue_hex.svg"
+import { HName, HUpper } from "../../../constants/typography"
+import { navyBlue, royalBlue } from "../../../constants/colors"
+import { size } from "../../../constants/devices"
+
+const TeamContainer = styled.div`
+  width: 86%;
+  margin: 0 auto;
+  padding-bottom: 16%;
+
+  @media only screen and (max-width: ${size.tablet}px) {
+    padding-bottom: 30vw;
+  }
+`
+
+const TeamPageTitle = styled(HUpper)`
+  color: ${royalBlue};
+  text-align: center;
+  margin: 60px 0 90px 0;
```

Comment:
From the design, the bottom margin for mobile could be reduced a little
<img src="https://user-images.githubusercontent.com/60420625/116790561-cf943880-aa82-11eb-83e9-837a701ca76b.jpeg" alt="team page Linh" width="300" />